laravel/app/Order.php

```diff
@@ -9,4 +9,11 @@ class Order extends Model
     protected $table = 'orders';
     protected $primaryKey = 'id';
     public $timestamps = false;
+
+    public function scopeOrderIndex($order_index)
+    {
+      	return $order_index
+    		->where('item_id', '=' ,$params['item_id'])
```

Comment:
`=` で判定する場合は `=` を記載する必要はないです！
```suggestion
    		->where('item_id', $params['item_id'])
```